README.md

```diff
@@ -1,26 +1,220 @@
+(_if you haven't already_)
+
+![circle-login-with-github](https://user-images.githubusercontent.com/194400/41608075-87c753b6-73df-11e8-910d-8ab2ebc2d21b.png)
+
+
+### 2. Add Project
+
+Click the "add project" `+` button from the left-hand side menu.
+Then select the project you want Circle-CI to test for you,
+and click the "**Setup Project**" button:
+
+![circle-add-project](https://user-images.githubusercontent.com/194400/41619549-950824e6-73fe-11e8-80de-390d56c90aff.png)
+
+You should see something like this now: <br />
+
+![circle-ci-setup-project](https://user-images.githubusercontent.com/194400/41620063-77108e54-7400-11e8-88c5-e434c7cb6474.png)
+
+
+### 3. Setup `.circleci/config.yml` File
+
+Next, follow the on-screen instructions
+to create a `config.yml` folder
+and setup your `.circleci/config.yml` file.
+```sh
+mkdir .circleci
+vi .circleci/config.yml
+```
+Click on the "***Copy To Clipboard***" button to get the code.
+_paste_ the sample code into the `.circleci/config.yml`
+file you have open in your text editor.
+
+e.g:
+
+![image](https://user-images.githubusercontent.com/194400/41620390-7937da24-7401-11e8-89a2-34a37cb7f130.png)
+
+Save the file, commit it and push to `GitHub`.
+e.g: https://github.com/nelsonic/circleci-hello-world-nodejs/commit/ad368321f3fa05686c97a8147f1e5570870bb527
+
+> **Note**: Circle-CI installs **Yarn** by default.
+This is ***NOT*** required anymore in 2018.
+It adds a _needless_ step to the build process and sends your `package.json`
+to Facebook HQ.
+I've left it as the `default` for this example, but we don't use Yarn
+for _any_ of our Node.js projects, there is ***zero benefit***.
+
+### 4. Start Building
+
+Once you have created the `.circleci/config.yml`,
+scroll down the page and click on "**Start building**":
+
+![circle-start-building](https://user-images.githubusercontent.com/194400/41621257-0945c61a-7404-11e8-86d3-9f206b509d49.png)
+
+You should be redirected to a build progress/status page:
+
+![circle-build-passed](https://user-images.githubusercontent.com/194400/41621398-6e1f49e4-7404-11e8-8d1a-5287bd47d2b5.png)
+
+e.g: https://circleci.com/gh/nelsonic/circleci-hello-world-nodejs/1
+
+The build and running the (_single unit_) test, took **3 seconds**
+and ended in "***SUCCESS***".
+(_it's a "hello world" example, it should be fast and pass_!)
+
+
+### 5. Status Badge!
+
+At this point we can add a "Badge" to our project `README.md`
+[![CircleCI](https://circleci.com/gh/nelsonic/circleci-hello-world-nodejs.svg?style=svg)](https://circleci.com/gh/nelsonic/circleci-hello-world-nodejs)
+
+e.g: https://circleci.com/gh/nelsonic/circleci-hello-world-nodejs/edit#badges
+![circle-ci-status-badge](https://user-images.githubusercontent.com/194400/41626981-46e652a2-7416-11e8-8e9e-0a03c8fde0a2.png)
+
+> For why this is a good idea, see: https://github.com/dwyl/repo-badges
+
+
+## Part 2 - Continuous _Deployment_
+
+Continuous Deployment (CD) is the perfect compliment
```

Comment:
`compliment`? Should it be `complement`

Reply:
Thanks for the _early_ review @SimonLab 😉

Reply:
Thanks for the compliment